### ISSUE TITLE
Jitter tests can fail

### ISSUE THREAD

**Issue by birdsarah** [bokeh/bokeh#4251]
The computed transforms use `Math.random()` which means the tests can fail

e.g. I just had the following failure

```sh
  1) Jitter transform module Jitter with normal should average the fixed values:

     AssertionError: expected 0.012654047923216893 to be below 0.01

    at Context.<anonymous> (test/models/transforms/jitter_transform.coffee:49:29)
```

I'm not sure if it's possible to seed Math.random or we need to set the tolerance on the test higher.

**Comment by damianavila** [comment]
I would use a seed if it is possible...

**Comment by bryevdv** [comment]
AFAIK there is no way to seed `Math.random` There are some alternative RNGs that offer this but then I'd be concerned about vetting their quality.

**Comment by justacec** [comment]
I was aware that this might happen when I made the test.  Perhaps it is best to remove the test.  I just wanted a way to verify that we were getting the actual normal distribution.

**Comment by birdsarah** [comment]
I'm doing a lot of JS testing at the moment. Total guess, but I would say it's failing half the time.

I have been learning a lot about sinon stubs though and may be able to stub Math.random(). I'll post an update.

Changing this to task.

**Comment by birdsarah** [comment]
I think I've cracked it. I'm going to keep testing.

**Comment by bryevdv** [comment]
@birdsarah if it's a simple fix early tomorrow, great, otherwise let's disable it for the time being to avoid jamming up other PRs tests.

**Comment by mattpap** [comment]
It's JS unit tests that are failing. Those are run in a node.js environment, where it should be possible to control seed (contrary to a web browser sandbox).

**Comment by birdsarah** [comment]
I'll submit a PR for the fix. I didn't use a seed, so @mattpap may have a different way of doing this.

**Comment by birdsarah** [comment]
So, there's a fix for this in #4203 - which is not idea, but I'm hoping that PR will get merged quickly.

If it doesn't, then I can break out.

**Comment by birdsarah** [comment]
This was fixed in #4203
